proxy/src/main/java/com/wavefront/agent/handlers/DeltaCounterAccumulationHandlerImpl.java

```diff
@@ -129,7 +136,18 @@ private void reportAggregatedDeltaValue(@Nullable HostMetricTagsPair hostMetricT
     if (reportedValue == 0) return;
     String strPoint = metricToLineData(hostMetricTagsPair.metric, reportedValue, Clock.now(),
         hostMetricTagsPair.getHost(), hostMetricTagsPair.getTags(), "wavefront-proxy");
-    getTask().add(strPoint);
+    getTask(APIContainer.CENTRAL_TENANT_NAME).add(strPoint);
+    // check if delta tag contains the tag key indicating this delta point should be multicasted
+    if (hostMetricTagsPair.getTags() != null && hostMetricTagsPair.getTags().containsKey(MULTICASTING_TENANT_TAG_KEY)) {
```

Comment:
Check first if `multicastingTenants` is active, it will  fast than `containsKey`

Reply:
Added this to Open Items in the design doc to track the progress on this.